### ISSUE TITLE
Can you give some details about training inception-resnet-v2??

### ISSUE THREAD

**Issue by YeonjaeShin** [soeaver/caffe-model#51]
Hi, I am trying to train faster-rcnn using inception-resnet-v2 with pascal voc2007 dataset.
I used your "caffe-model/det/faster_rcnn/models/pascal_voc/solver.prototxt" and "caffe-model/det/faster_rcnn/models/pascal_voc/inception-resnet-v2/faster_voc_inception-resnet-v2-merge-aligned.prototxt" to train.

I changed your solver.prototxt like following.
-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
base_lr: 0.045
lr_policy: "step"
gamma: 0.94
stepvalue: 50000
display: 20
average_loss: 100
momentum: 0.9
weight_decay: 0.0001
snapshot: 0
snapshot_prefix: "faster_inception_resnet-v2"
iter_size: 1
-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
When I ran the command : "time ./tools/train_net.py --gpu 0 --solver models/pascal_voc/Inception-resnet-v2/faster_rcnn_end2end/solver.prototxt --weights data/imagenet_models/inception-resnet-v2.caffemodel --imdb voc_2007_trainval --iters 80000 --cfg experiments/cfgs/faster_rcnn_end2end.yml"

I got this error message...
-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
/home/yeonjae/py-faster-rcnn/tools/../lib/fast_rcnn/bbox_transform.py:48: RuntimeWarning: overflow encountered in exp
  pred_w = np.exp(dw) * widths[:, np.newaxis]
/home/yeonjae/py-faster-rcnn/tools/../lib/fast_rcnn/bbox_transform.py:48: RuntimeWarning: overflow encountered in multiply
  pred_w = np.exp(dw) * widths[:, np.newaxis]
/home/yeonjae/py-faster-rcnn/tools/../lib/fast_rcnn/bbox_transform.py:49: RuntimeWarning: overflow encountered in exp
  pred_h = np.exp(dh) * heights[:, np.newaxis]
/home/yeonjae/py-faster-rcnn/tools/../lib/fast_rcnn/bbox_transform.py:49: RuntimeWarning: overflow encountered in multiply
  pred_h = np.exp(dh) * heights[:, np.newaxis]
I1011 18:06:21.929924  3081 solver.cpp:218] Iteration 0 (0 iter/s, 0.894591s/20 iters), loss = 3486.56
I1011 18:06:21.930017  3081 solver.cpp:237]     Train net output #0: loss_bbox = 3265.53 (* 1 = 3265.53 loss)
I1011 18:06:21.930033  3081 solver.cpp:237]     Train net output #1: loss_cls = 87.3365 (* 1 = 87.3365 loss)
I1011 18:06:21.930045  3081 solver.cpp:237]     Train net output #2: rpn_cls_loss = 44.6777 (* 1 = 44.6777 loss)
I1011 18:06:21.930058  3081 solver.cpp:237]     Train net output #3: rpn_loss_bbox = 89.0116 (* 1 = 89.0116 loss)
Floating point exception (core dumped)
-- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- 
Have you seen this error?? And could you please share your settings or things you used when training inception-resnet-v2??
Thanks~!

**Comment by soeaver** [comment]
Sorry, the inrv2 model is converted from tf, I didn't train the model from scrach.

**Comment by YeonjaeShin** [comment]
Oh...then just to make sure, is it right to use your "caffe-model/det/faster_rcnn/models/pascal_voc/inception-resnet-v2/faster_voc_inception-resnet-v2-merge-aligned.prototxt" to train 
and  "caffe-model/det/faster_rcnn/models/pascal_voc/inception-resnet-v2/deploy_faster_voc_inception-resnet-v2-merge-aligned.prototxt to test?

**Comment by soeaver** [comment]
yes

**Comment by YeonjaeShin** [comment]
Sorry for bothering..
But in the other issue, I saw you uploaded your model here.."https://pan.baidu.com/s/1jHPJCX4#list/path=%2F"
I am little bit confused because there are several prototxts and I am not sure which one to use.
Could you explain some difference of thoses prototxts?

**Comment by soeaver** [comment]
Difference prototxts for  difference task, cls/det/seg.
